docker/joex-entrypoint.sh

```diff
@@ -3,4 +3,15 @@
 echo "Starting unoconv listener"
 unoconv -l &
 
+# replace own ocrmypdf with official dockerfile, i.e. newer version
+if [ -S "/var/run/docker.sock" ]; then
+  echo "Found 'docker.sock': Installing Docker and redirecting 'ocrmypdf' command to official dockerfile by jbarlow83"
+  apk --no-cache add docker
+  docker pull -q jbarlow83/ocrmypdf:$OCRMYPDF_VERSION
+  function ocrmypdf () {
```

Comment:
This is really interesting. this function is enough to declare such that it is in the `PATH` and docspell can call it?

Reply:
Ah it is in the environment of the parent process right? It's not necessary to `export` it? (sorry, this is not a critique I'm just curious)

Reply:
I never take your questions as critique, plus usually they are totally important!

As in this case - i only tested in interactively, and did not put enough attention to it when testing in the final environment.

In the end, I had to move from alias to bash function now to script in `/usr/local/bin` which comes before `/usr/bin` where the original ocrmypdf executable is